### ISSUE TITLE
Default theme colors are not accessible

### ISSUE THREAD

**Issue by chadlavi** [TypeStrong/typedoc#1369]
The colors in the default theme do not make an accessible color contrast on a white background. The green link color (`#7da01f`) has a color contrast of 3.03. Minimum WCAG AA compliance for small text is 4.5.

cf. https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html

## Search terms

accessibility, a11y, color, color contrast

## Expected Behavior

All websites should strive to offer an experience accessible to all users.

## Actual Behavior

The color contrasts in the default themes provided by typedoc are not accessible.

**Comment by Gerrit0** [comment]
PR welcome - https://github.com/TypeStrong/typedoc-default-themes/blob/master/src/default/assets/css/_constants.sass

**Comment by chadlavi** [comment]
@Gerrit0 I don't mind taking a look! People are often touchy about theme colors, so I didn't want to just barrel in with a change in some of the fundamental look and feel of the product.

**Comment by Gerrit0** [comment]
This has been fixed in version the themes version 0.11.4. If you delete your project's lockfile and reinstall, the newer version should be picked up automatically.
